Split/Common/ServiceConstants.swift

```diff
@@ -17,4 +17,5 @@ struct ServiceConstants {
     static let impressionsQueueSize: Int = 30000
     static let defaultDataFolder = "split_data"
     static let cacheExpirationInSeconds = 864000
+    static let controlNoCacheHeader = [ServiceConstants.CacheControlHeader: ServiceConstants.CacheControlNoCache]
```

Comment:
I don't see (at least in this PR) that we are using this header also for `/splitChanges` requests due to SPLIT_UPDATE and SPLIT_KILL push notifications. If it is in another branch, recheck Javi before release, and if possible, add some asserts in streaming tests to check that the header is properly set.